app/models/course_user.rb

```diff
@@ -2,4 +2,9 @@ class CourseUser < ActiveRecord::Base
   belongs_to :user, inverse_of: :course_users
   belongs_to :course, inverse_of: :course_users
   enum role: { student: 0, teaching_assistant: 1, manager: 2, owner: 3 }
+  scope :student, -> { where(role: roles[:student]) }
```

Comment:
Given that we can get scope like behaviour by .<role> do we still want the scope? Well enum does not support plural form.